src/LinuxTracing/UprobesUnwindingVisitor.cpp

```diff
@@ -430,6 +438,16 @@ void UprobesUnwindingVisitor::Visit(uint64_t event_timestamp,
             /*registers=*/std::nullopt, event_data.function_id);
 }
 
+void UprobesUnwindingVisitor::Visit(uint64_t /*event_timestamp*/,
```

Comment:
Please move it outside of the `UprobesPerfEventData`, `UprobesWithArgumentsPerfEventData`, `UretprobesPerfEventData`, `UretprobesWithReturnValuePerfEventData`, `UserSpaceFunctionEntryPerfEventData`,  `UserSpaceFunctionExitPerfEventData` group, as these work together for dynamic instrumentation, while `Visit` for `UprobesWithStackPerfEventData` has a different purpose, so it's a bit distracting when going through the file. Before `UprobesPerfEventData` or after  `UserSpaceFunctionExitPerfEventData` should be fine.